src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1542,7 +1464,7 @@
 
 helps['network route-filter'] = """
     type: group
-    short-summary: (PREVIEW) Manage route filters.
+    short-summary: (Preview) Manage route filters.
```

Comment:
Recommend choosing one approach for preview commands. Either (Preview) or the long-summary "These commands are in preview".

Reply:
Recommendation from @JasonRShaver would be appreciated here. I'm not sure how the CLI team would prefer to communicate preview/experimental commands; It depends on the level of importance of informing the user.

Reply:
(Preview)

Additionally, we can put more details in the long-summary (this command requires joining a private beta, please contact blah@microsoft.com for more information) if that adds value.

Reply:
Can you email me a list of preview commands which require special access, and the contact emails needed for each?